Gruntfile.js

```diff
@@ -29,6 +29,14 @@ module.exports = function (grunt) {
       }
     }()),
 
+    i18nextract: {
+      default_options: {
+        src: [ 'src/**/*.js', 'src/**/*.html' ],
+        lang: ['en'],
+        dest: 'build/tmp'
```

Comment:
the build folder's all temp so maybe the destination can be renamed, but otherwise I think it's a good spot.  It get's cleaned up at the beginning of a build, https://github.com/elastic/kibana/blob/master/tasks/build/index.js#L6.